packages/circuit-ui/components/Tag/Tag.tsx

```diff
@@ -171,6 +171,7 @@ const closeButtonStyles = ({ theme }: StyleProps) => css`
   right: ${BORDER_WIDTH};
   transform: translateY(-50%);
   border-radius: ${theme.borderRadius.byte};
+  border: 0;
```

Comment:
Why are we changing this? 🙂

Reply:
It's related to the changes in the CloseButton. It was two pixels shorter than other buttons because it had the border removed. I made the border transparent instead, but that broke the remove button in the Tag. Here, we actually want the button to be slightly shorter so it fits inside the Tag

Reply:
:mildpanic:

👍